.nvmrc

```diff
@@ -1 +1 @@
-18
\ No newline at end of file
+lts/*
```

Comment:
@tharropoulos I'd prefer that we be explicit about the node version we're using, so as to not accidentally end up dropping support for an older version without intending to in the future. Because we also need to document any Node compatibility changes in the changelogs.